inc/Addon/WebP/Subscriber.php

```diff
@@ -118,7 +118,7 @@ public function convert_to_webp( $html ) {
 			return $html . '<!-- Rocket no webp -->';
 		}
 
-		if ( ! isset( $this->filesystem ) ) {
+		if ( ! isset( $this->filesystem ) ) { // @phpstan-ignore-line
```

Comment:
`Property WP_Rocket\Addon\WebP\Subscriber::$filesystem (WP_Filesystem_Direct) in isset() is not nullable` 
But as we are not setting it in the constructor, I was thinking it could be null. What do you think ?

Reply:
We could declare it as nullable instead of ignoring the error?